GA - Updating Authentication Methods Policy V1.0 branch with new feature settings changes. (#17821)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/featuretarget.md

---
title: "featureTarget resource type"
description: "Defines a single group, Azure AD role, or administrative unit that is included or excluded in the settings specified in the authenticationMethodFeatureConfiguration object."
author: "mjsantani"
ms.localizationpriority: medium
ms.prod: "identity-and-sign-in"
doc_type: resourcePageType
---

# featureTarget resource type

Namespace: microsoft.graph

Defines a single group, Azure AD role, or administrative unit that is included or excluded in the settings specified in the [authenticationMethodFeatureConfiguration](authenticationmethodfeatureconfiguration.md) object.

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |id|String|The ID of the entity that's targeted in the include or exclude rule. Additional possible value is: `all_users`.|

Reviewer: ```suggestion
|id|String|The ID of the entity that's targeted in the include or exclude rule, or `all_users` to target all users.|
```
